spec.html

```diff
@@ -13844,7 +13844,7 @@ <h1>
       </h1>
       <dl class="header">
         <dt>for</dt>
-        <dd>a built-in function object _F_</dd>
+        <dd>a built-in function object _F_ (when present)</dd>
```

Comment:
```suggestion
        <dd>a built-in function object _F_ (when the method is present)</dd>
```
maybe? But I'm OK either way.